src/users/music/musics.service.ts

```diff
@@ -55,6 +58,7 @@ export class MusicsService {
       const album = await this.playlistsService.getPlaylistInfo(
         new Types.ObjectId(uploadMusicDto.albumId),
       );
+      if (album.isAlbum === false) throw new BadRequestException();
     } catch (err) {
```

Comment:
Did you forget the message in the error?

Reply:
No, since no matter error message in this error will soon be overwrite by this [line](https://github.com/2110336-2565-2/sec2-group10-demo-backend/blob/db24fa75258ecfc8ff0663b19058bd77d1c7bc0f/src/users/music/musics.service.ts#L63), I see it unnecessary.